tools/llcstat.py

```diff
@@ -41,16 +46,19 @@
 
 struct key_t {
     int cpu;
-    int pid;
+    u32 pid;
+    TIDINFO
```

Comment:
Can we just keep `u32 tid` here for both use-tid and dont-use-tid cases? I'd like to limit use of `bpf_text.replace` as it makes the program harder to understand